src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    public Driver create(Driver driver);
+
+    public Optional<Driver> get(Long id);
+
+    public List<Driver> getAll();
+
+    public Driver update(Driver driver);
+
+    public boolean delete(Long id);
```

Comment:
methods in interface are public by default
let's remove public keyword from methods

Reply:
refactored